content/src/instrument.ts

```diff
@@ -1,42 +1,4 @@
 import { ILoggerComponent } from '@well-known-components/interfaces'
```

Comment:
Is this file being used now?

Reply:
yeah, there's still one remaining function